### DISCUSSION TITLE
Setup fails in AWS with bridge interface

### DISCUSSION THREAD

**Discussion by Stephen-Ridgway** [Security-Onion-Solutions/securityonion#3401] [Unsupported Versions]
Hi all,

I have an AWS server with one interface and a bridge group. I am trying to create an Eval SO. The network config is:

ubuntu@ip-172-31-1-5:~/securityonion$ ifconfig
br0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
        inet 172.31.1.5  netmask 255.255.255.0  broadcast 172.31.1.255
        inet6 fe80::430:73ff:fe3e:f8be  prefixlen 64  scopeid 0x20<link>
        ether 06:30:73:3e:f8:be  txqueuelen 1000  (Ethernet)
        RX packets 2372  bytes 19467685 (19.4 MB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1829  bytes 200091 (200.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
        ether 06:30:73:3e:f8:be  txqueuelen 1000  (Ethernet)
        RX packets 41603  bytes 56633512 (56.6 MB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 7215  bytes 789522 (789.5 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 286  bytes 26272 (26.2 KB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 286  bytes 26272 (26.2 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

The Netplan yaml file for this configuration is:

![image](https://user-images.githubusercontent.com/80473697/110827803-551f1780-828e-11eb-830f-449c28b38f23.png)

The first error I get is that SO setup only thinks there is one interface:
![image](https://user-images.githubusercontent.com/80473697/110828232-b9da7200-828e-11eb-8984-556b2811eed2.png)

The next error is that the interface is not properly recognised anyway:
![image](https://user-images.githubusercontent.com/80473697/110828381-e4c4c600-828e-11eb-9e3b-438b3a36abe1.png)

Bridge utils shows the bridge as configured and seems to be working (I can connect to the server, at least):
ubuntu@ip-172-31-1-5:~/securityonion/securityonion$ sudo brctl show
bridge name	bridge id		STP enabled	interfaces
br0		8000.0630733ef8be	no		eth0

This config worked in SO 16.04

Any help gratefully received.

Stephen

**Comment by weslambert** [comment]
I have not tested with a bridge interface, but can confirm other interfaces work as intended.
